api/woeip/apps/air_quality/serializers.py

```diff
@@ -73,7 +73,7 @@ def create(self, validated_data):
             collection_file.file.save(
                 upload_file["file_name"], ContentFile(upload_file["file_data"]))
 
-        return super().create(validated_data)
+        return collection
```

Comment:
This looks good. I was curious to see what else is meant to happen in the call to the base class `create` (since `HyperlinkedModelSerializer` doesn't implement `create` this ends up being `ModelSerializer.create`). I found the docstring to be interesting:

https://github.com/encode/django-rest-framework/blob/30e56f62ba1115ce23851995c6ed6bff45defbb7/rest_framework/serializers.py#L911

Long story short, what we're doing here is :+1: